### PR TITLE
Created online shopping

### PR DESCRIPTION
**Pull request by TuyishimireEric**
# React capstone project - Online shopping
This project is based on an eCommerce website built with React and Redux. The user can check all the products in our and get more details on selected products. The user can also search for a product in our store.

## Highlights
API First I have chosen a fakeStore API to base the development of the web app on. The API allows getting numeric values based on a parameter. and the chosen API is open. 

### Design
image

### Technical 
- [x] The project is a single-page application (SPA) built with React and Redux.
- [x] The data retrieved from the API should be stored in the Redux store.
- [x] I filtered the data that you retrieve from the API using a Filter stateless component.
- [x] Every page (the main page and the pages for each item) has a unique route within the SPA.
- [x] The project is deployed and accessible online. [project link](https://et-online-shopping.netlify.app/)

**Cheers and happy reviewing!**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,76 @@
...
+## 🤝 Contributing
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+Contributions, issues, and feature requests are welcome!
 
-### Analyzing the Bundle Size
+Feel free to check the [issues page](https://github.com/29td/online-shopping/issues).
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+## Acknowledgments
```
Comment (AnselemOdims) [inline README.md:70]: - [ ] Please kindly give appropriate credit to the author of the original design.

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @TuyishimireEric , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please kindly highlight a piece of code the find interesting or proud of making
- [ ] Please kindly include integration tests using React Testing Library
- [ ] Good job on personalizing your design but please ensure it looks close to the original design. The cards should be the same height and touch each other like the design.
- [ ] Please also ensure you stick to the design guidelines 
    - Typography: font face, size, and weight
    - Layout: composition and space between elements.
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 1 above)
Comment (TuyishimireEric) [reply README.md:70]: Thank you @AnselemOdims for reviewing my project!

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @TuyishimireEric ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions

##### _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

##### Cheers and Happy coding!:clap::clap::clap:
##### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by TuyishimireEric** [comment]
Thank you @1GORDON for approving my project!

**Pull request merged**